account_move_line_stock_info/tests/test_account_move_line_stock_move.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests import common
+
+
+class TestAccountMoveLineStockInfo(common.TransactionCase):
+
+    def setUp(self):
+        super(TestAccountMoveLineStockInfo, self).setUp()
+        self.product_model = self.env['product.product']
+        self.product_ctg_model = self.env['product.category']
+        self.acc_type_model = self.env['account.account.type']
+        self.account_model = self.env['account.account']
+        self.aml_model = self.env['account.move.line']
+        self.stock_picking_model = self.env['stock.picking']
+        self.stock_move_model = self.env['stock.move']
+        self.res_users_model = self.env['res.users']
+
+        self.partner1 = self.env.ref('base.res_partner_1')
+        self.location_stock = self.env.ref('stock.stock_location_stock')
+        self.location_supplier = self.env.ref('stock.stock_location_suppliers')
+        self.location_customer = self.env.ref('stock.stock_location_customers')
+        self.company = self.env.ref('base.main_company')
+        self.picking_type_in = self.env.ref('stock.picking_type_in')
+        self.picking_type_out = self.env.ref('stock.picking_type_out')
+        self.group_stock_user = self.env.ref('stock.group_stock_user')
+        self.group_account_invoice = self.env.ref(
+            'account.group_account_invoice')
+        self.group_account_manager = self.env.ref(
+            'account.group_account_manager')
+
+        # Create account for Goods Received Not Invoiced
+        acc_type = 'equity'
+        name = 'Goods Received Not Invoiced'
+        code = 'grni'
+        self.account_grni = self._create_account(acc_type, name, code,
+                                                 self.company)
+
+        # Create account for Cost of Goods Sold
+        acc_type = 'expense'
+        name = 'Cost of Goods Sold'
+        code = 'cogs'
+        self.account_cogs = self._create_account(acc_type, name, code,
+                                                 self.company)
+        # Create account for Inventory
+        acc_type = 'asset'
+        name = 'Inventory'
+        code = 'inventory'
+        self.account_inventory = self._create_account(acc_type, name, code,
+                                                      self.company)
+        # Create Product
+        self.product = self._create_product()
+        # company
+
+        # Create users
+        self.stock_user = self._create_user('stock_user',
+                                            [self.group_stock_user,
+                                             self.group_account_invoice],
+                                            self.company)
+        self.account_invoice = self._create_user('account_invoice',
```

Comment:
Maybe something like 'accountant' or 'account_user' will be more descriptive, do you agree?